playbooks/common/openshift-cluster/upgrades/pre/verify_cluster.yml

```diff
@@ -45,13 +45,18 @@
 
   # Need client package to check gluster health when upgrading from containerized.
   - name: Install openshift clients on first master
-    package:
-      name: "{{ openshift_service_type }}-clients{{ (openshift_pkg_version | default('')) | lib_utils_oo_image_tag_to_rpm_version(include_dash=True) }}"
-      state: present
+    command: "{{ ansible_pkg_mgr }} install -y {{ openshift_node_upgrade_rpm_list | join(' ')}}"
     register: result
     until: result is succeeded
-    retries: 3
-    when: not openshift_is_atomic | bool
```

Comment:
The task will break on Atomic without this

Reply:
Woops. I did not mean to remove that. Fixed.